layouts/home.ejs

```diff
@@ -103,8 +103,8 @@
       </header>
       <div class="sponsors__content">
         <div class="sponsors__link">
-          <a href="https://tutanota.com/" target="_blank">
-            <img src="/static/tutanota-logo.webp" alt="tutanota.com" class="sponsors__logo">
+          <a href="https://tuta.com/" target="_blank">
+            <img src="/static/tuta_logo.svg" alt="tuta.com" class="sponsors__logo">
```

Comment:
We might want to specify an explicit width here, since SVGs don't have intrinsic width or height. Let me check it locally to see how it looks.

Reply:
Yeah, it's _huge_. Give me a minute to fix this.